code/modules/shuttle/shuttle_rotate.dm

```diff
@@ -83,3 +87,10 @@ If ever any of these procs are useful for non-shuttles, rename it to proc/rotate
 		var/new_pos = supposed_node_connect.Find(real_node_connect[i])
 		nodes[new_pos] = nodes_copy[i]
 
+/obj/machinery/light/shuttleRotate(rotation, params)
+	params = ROTATE_DIR	//Do not rotate offsets, light fixtures already handle them when changing direction
+	return ..()
+
+/obj/machinery/computer/camera_advanced/shuttle_docker/flyable/shuttleRotate(rotation, params)
+	params = ROTATE_DIR
+	return ..()
```

Comment:
?

Reply:
??? Do you want stuff to not change direction when the shuttle rotates?